### ISSUE TITLE
Styling issues in console (Python)

### ISSUE THREAD

**Issue by JasonWeill** [jupyterlab/jupyterlab#14541]
In Notebook 7.0.0a17 and JupyterLab 4.0.0rc1, the console (Python) has some weird styling issues.

- [ ] If you click on a cell, you initiate a drag interaction but you can't drag cells.
- [ ] No favicon for terminal in browser tab.
- [ ] every run cell gets a blinking text cursor. 
- [ ] thin toolbar at top of window.

![Screen Shot 2023-03-20 at 3 42 50 PM](https://user-images.githubusercontent.com/93281816/226791909-5dcaebcf-2ecf-4d19-a547-f532641fd19c.png)

This was found by @ellisonbg during a bug bash, and moved from https://github.com/jupyter/notebook/issues/6789 per @jtpio.

**Comment by krassowski** [comment]
>  If you click on a cell, you initiate a drag interaction but you can't drag cells.

This is incorrect. You can drag cells from console into a notebook. This is a Feature.

![DRAG](https://github.com/jupyterlab/jupyterlab/assets/5832902/7b60d71c-a852-439f-a5b3-32d9e05b4589)

>  No favicon for terminal in browser tab.

This is not a bug - it was never implemented in JupyterLab AFAIK. Can you open a dedicated issue?

>  every run cell gets a blinking text cursor.
>  thin toolbar at top of window.

Both were fixed in https://github.com/jupyterlab/jupyterlab/pull/15779

**Comment by krassowski** [comment]
Closing as everything works as expected now - the favicon request can be tracked in a dedicated issue with a more appropriate title.
